updater/lib/dependabot/setup.rb

```diff
@@ -8,6 +8,7 @@
 require "dependabot/logger/formats"
 require "dependabot/opentelemetry"
 require "dependabot/sentry"
+require "dependabot/service"
```

Comment:
🔎 I'm curious why this was needed. Was it missing before?

Reply:
Oh no, not needed just an oversight from something else I tried, let me clean it up :+1: